src/main/java/de/craften/ui/swingmaterial/MaterialComboBox.java

```diff
@@ -89,7 +153,7 @@ public void paint(Graphics g) {
 
             g.setFont(Roboto.REGULAR.deriveFont(15f));
             if (selected) {
-                g2.setColor(MaterialColor.PINK_500);
+                g2.setColor(comboBox.accentColor);
             } else {
                 g2.setColor(Color.BLACK);
```

Comment:
```java
public static class FieldRenderer<T> extends JComponent implements ListCellRenderer<T> {
```

The line you mention is using the accent color for the selected item in the list (actually taking place of `MaterialColor.PINK_500`).

Although now I see the `FieldRenderer` is still kind of using a fixed set of colors instead of the `MaterialComboBox`'s foreground and background. I think this should be addressed...